readme.md

```diff
@@ -0,0 +1,23 @@
+How to build
+------------
+
+Things you'll need to develop and test abracadabra:
+
+* [Node.js and npm](https://nodejs.org/en/download/)
+* [Yarn](https://yarnpkg.com/en/docs/install)
+
+Clone the repository and then at the root of your local copy install the dependencies with yarn:
+
+	yarn
+
+Now everything should be good to go.
+
+You can execute tests with the command
+
+	npm test
```

Comment:
Considering the dependency installation is made with `yarn`, it would make sense to also run the commands with yarn, e.g. `yarn test` and `yarn start`. Or, then use npm throughout, and replace the first command with `npm install`.

I'd personally maybe suggest going with yarn throughout.

Reply:
Ah, I had not realised yarn can also substitute that part of npm in addition to installing packages. I then think it better to not mention npm at all.